packages/page-claims/src/Statement.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2020 @polkadot/app-claims authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { StatementKind } from '@polkadot/types/interfaces';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import PolkadotRegular from './statements/polkadot/regular';
+import PolkadotSaft from './statements/polkadot/saft';
+import { useTranslation } from './translate';
+import { getStatement } from './util';
+
+export interface Props {
+  className?: string;
+  kind?: StatementKind;
+  systemChain: string;
+}
+
+// Get the full hardcoded text for a statement
+function StatementFullText ({ kind, systemChain }: { kind?: StatementKind; systemChain: string }): React.ReactElement | null {
+  const { t } = useTranslation();
+
+  switch (systemChain) {
+    case 'Polkadot CC1': {
+      if (!kind) {
+        return null;
+      }
+
+      return kind.isRegular
+        ? <PolkadotRegular />
+        : <PolkadotSaft />;
+    }
+
+    default:
+      return <p>{t('Warning: we did not find any attest statement for {{chain}}', { replace: { chain: systemChain } })}</p>;
+  }
+}
+
+function Statement ({ className, kind, systemChain }: Props): React.ReactElement<Props> | null {
+  const { t } = useTranslation();
+  const statementUrl = getStatement(systemChain, kind)?.url;
+
+  return (
+    <div className={className}>
+      {t('Please read these terms and conditions carefully. By submitting this statement, you are deemed to have accepted these Terms and Conditions. If you do not agree to these terms, please refrain from accessing or proceeding. You can also find them at:')}
```

Comment:
```suggestion
      {t('Please read these terms and conditions carefully. By submitting this statement, you are deemed to have accepted these Terms and Conditions. If you do not agree to these terms, please refrain from accessing or proceeding. You can also find them at: ')}
```

A space here prevents the "at" from being attached to the link like this:

![image](https://user-images.githubusercontent.com/25539605/82819071-10152a80-9ea0-11ea-83be-4cbea349eba2.png)

Putting the space fixes it:

![image](https://user-images.githubusercontent.com/25539605/82819242-5e2a2e00-9ea0-11ea-98a4-e24fe36e7718.png)